pkg/node/devnode.go

```diff
@@ -112,7 +112,10 @@ func NewDevBee(logger log.Logger, o *DevOptions) (b *DevBee, err error) {
 	}
 	b.stateStoreCloser = stateStore
 
+	post := mockPost.New()
```

Comment:
This is not required. We dont have to set any expiry handler. If it is nil, we already handle the case.